handbook/people-ops/onboarding/index.md

```diff
@@ -13,6 +13,7 @@
 
 - [ ] Add yourself to the [team page](../../../company/team/index.md).
   - [ ] Once you've created a team page entry, add the link to your bio in your Slack profile 
+- [ ] Add your geo location to the [team map](../../../company/team/locations.geojson).
```

Comment:
There's already a note on the template: https://github.com/sourcegraph/about/blob/master/company/team/index.md#full-name-optional-pronouns

Might not hurt to have more reminders though 🤔